scripts/eurostat/health_determinants/common/euro_stat.py

```diff
@@ -0,0 +1,343 @@
+            'isced97': 'isced11',
+            'quantile': 'quant_inc'
+        },
+                  inplace=True)
+
+        df = self._rename_frequency_column(df)
+
+        if 'quant_inc' in df.columns.values.tolist():
+            df = df[(~(df['quant_inc'] == 'UNK'))]
+
+        df = df[df['age'] == 'TOTAL']
+        df = replace_col_values(df)
+
+        if file_name in file_to_sv_mapping[import_name]:
+            df['SV'] = eval(file_to_sv_mapping[import_name][file_name])
+        else:
+            print(
+                '#########\nERROR: File (', file_name,
+                ') to SV mapping missing',
+                '\nAdd a File - SV mapping statement in common/sv_config.py.',
+                '\n#########')
+            exit(1)
+
+        del_columns = list(df.columns.difference(original_df_columns))
+        if 'time' in del_columns:
+            del_columns.remove('time')
+            id_vars = ['SV', 'time']
+            var_name = 'geo'
+        elif 'geo' in del_columns:
+            del_columns.remove('geo')
+            id_vars = ['SV', 'geo']
+            var_name = 'time'
+        del_columns.remove('SV')
+
+        df.drop(columns=del_columns, inplace=True)
+
+        df = df.melt(id_vars=id_vars,
+                     var_name=var_name,
+                     value_name='observation')
+        df = df[df['geo'] != 'EU27_2020']
+        df = df[df['geo'] != 'EU28']
+        return df
+
+    def generate_csv(self) -> pd.DataFrame:
+        """
+        This Method calls the required methods to generate
+        cleaned CSV, MCF, and TMCF file.
+
+        Args:
+            None
+
+        Returns:
+            df (pd.DataFrame)
+        """
+        final_df = pd.DataFrame(
+            columns=['time', 'geo', 'SV', 'observation', 'Measurement_Method'])
+        # Creating Output Directory
+        output_path = os.path.dirname(self._cleaned_csv_file_path)
+        if not os.path.exists(output_path):
+            os.mkdir(output_path)
+
+        dfs = []
+        for file_path in self._input_files:
+            df = pd.read_csv(file_path, sep='\t', header=0)
+            file_name = file_path.split("/")[-1][:-4]
+            df.columns = df.columns.str.strip()
+            df = self._parse_file(file_name, df, self._import_name)
+            df['SV'] = df['SV'].str.replace('_Total', '')
+            dfs.append(df)
+        final_df = pd.concat(dfs, axis=0)
+
+        final_df = final_df.sort_values(by=['time', 'geo', 'SV', 'observation'])
+        final_df = final_df.drop_duplicates(subset=['time', 'geo', 'SV'],
+                                            keep='first')
+        final_df['observation'] = final_df['observation'].astype(
+            str).str.strip()
+        # derived_df generated to get the year/SV/location sets
+        # where 'u' exist
+        derived_df = final_df[final_df['observation'].astype(str).str.contains(
+            'u')]
+        u_rows = list(derived_df['SV'] + derived_df['geo'])
```

Comment:
The earlier question still remains:
Shouldn't 'info' contain the `'time'` column as well assuming for the same place+SV, some year observations could be good and some year observations low-reliability?

Reply:
Discussed offline that the mmethod for a country+statvar will be set to a constant (LowReliability/DefinitionDiffers) so the timeseries has no date holes/splits.